scip-semanticdb/src/main/java/com/sourcegraph/scip_semanticdb/BazelBuildTool.java

```diff
@@ -65,7 +65,9 @@ public boolean hasErrors() {
             mavenPackages,
             /* buildKind */ "",
             /* emitInverseRelationships */ true,
-            /* allowEmptyIndex */ true);
+            /* allowEmptyIndex */ true,
+            /* indexDirectoryEntries */ false // because Bazel only compiles to jar files.
```

Comment:
nit (for future refactor): this many boolean params in a row is a bit annoying, and we can't use named parameters with java class

At some point I think we should generate a builder for this options class to make things clearer.

Reply:
Agreed. Builders are great when using Java.

Reply:
We can't used named arguments from Scala if the definition is in Java.

Reply:
The problem with builders is that we can't make the fields final, but it's not a problem here. We could simplify a lot by just providing sensible default values